src/lib/Router/AppRouter.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes
+} from 'react-router-dom';
+import { Link as RouterLink, LinkProps } from 'react-router-dom';
+import { NotFound } from '../../pages';
+
+type LinkFacadeProps = LinkProps & {
+  // add any other props you want to pass to the Link component
```

Comment:
@ISHIMWE-Aime leave an example here to allow others to have an idea of how to use this. or document it above.